src/lib/LR1121Driver/LR1121.cpp

```diff
@@ -126,6 +126,14 @@ transitioning from FS mode and the other from Standby mode. This causes the tx d
     }
 #endif
 
+    // 2.1.3.1 CalibImage
+    uint8_t CalImagebuf[2];
+    CalImagebuf[0] = ((minimumFrequency / 1000000 ) - 1) / 4;       // Freq1 = floor( (fmin_mhz - 1)/4)
+    CalImagebuf[1] = 1 + ((maximumFrequency / 1000000 ) + 1) / 4;   // Freq2 = ceil( (fmax_mhz + 1)/4)
+    hal.WriteCommand(LR11XX_SYSTEM_CALIBRATE_IMAGE_OC, CalImagebuf, sizeof(CalImagebuf), SX12XX_Radio_All);
+    DBGLN("CalibImage minimum frequency: %d", CalImagebuf[0]);
```

Comment:
This doesn't seem like something we'd ever need to troubleshoot, is the DBG necessary?